test/perf/react.js

```diff
@@ -94,31 +95,33 @@ describe('update', () => {
   it('skate', done => {
     fixture.innerHTML = '<x-app></x-app>';
 
-    let comp = fixture.firstElementChild;
+    const comp = fixture.firstElementChild;
     ready(comp, () => {
-      bp(function (next) {
+      bp(function (next) { // eslint-disable-line func-names
         props(this.comp, { title: ++this.count });
         next();
       }, {
         comp,
-        count: 0
+        count: 0,
       })
-        .then(ops => console.log(`Skate (update): ${ops} / sec`))
-        .then(() => fixture.innerHTML = '')
+        .then(ops => console.log(`Skate (update): ${ops} / sec`)) // eslint-disable-line no-console
+        .then(() => {
+          fixture.innerHTML = '';
+        })
```

Comment:
You can also wrap the assignment in parenthesis instead of having to use curly braces.